src/cromshell/utilities/io_utils.py

```diff
@@ -101,6 +103,59 @@ def assert_path_is_not_empty(path: Union[str, Path], description: str) -> None:
             raise EOFError(f"ERROR: {description} is empty: {path}.")
 
 
+def has_nested_dependencies(wdl_path: str or Path) -> bool:
+    """Determine if a WDL has any nested imports."""
+
+    with open(wdl_path, 'r') as rf:
+        for l in rf:
+            if l.startswith('import'):
+                m = re.match(r'import "(.+)"', l)
+
+                if "../" in m.group(1):
```

Comment:
An edge case that this function would miss would be nested wdl that uses absolute paths for their imports. Highly unlikely though as I haven't seen wdls like that often or at all.